web/gds-user-ui/src/locales/en/messages.po

```diff
@@ -95,7 +114,7 @@ msgstr "Action"
 msgid "Add Address"
 msgstr "Add Address"
 
-#: src/components/OtherJuridictions/index.tsx:61
+#: src/components/OtherJurisdictions/index.tsx:61
```

Comment:
Unfortunately this is a typo in a filepath name that we will have to live with for now - changing it here would not change the filepath and would make it impossible for the program to find the translated content that below there. Would you mind changing it back?

Reply:
Understood, I'll change it back.